src/main/java/net/neoforged/neoforge/event/AddPackFindersEvent.java

```diff
@@ -44,4 +46,11 @@ public void addRepositorySource(RepositorySource source) {
     public PackType getPackType() {
         return packType;
     }
+
+    /**
+     * {@return whether or not the pack repository being assembled is "trusted" to provide default KnownPacks}
+     */
+    public boolean isTrusted() {
+        return trusted;
+    }
```

Comment:
What does being "trusted" entail here? When should a mod check this to do something differently?

Reply:
Basically: 9 times out of 10 modders can ignore this. It's here in case a modder, for some reason, wants to add a pack with a KnownPack that is not discovered by the client in sync - or, in general, wants to have different logic for the client-only pack being used to provide known packs than for the "normal" pack. The name comes from what vanilla calls it.

Reply:
Added some better docs